api-catalog-ui/frontend/src/components/App/_app.scss

```diff
@@ -184,53 +185,53 @@ body {
       margin-right: var( --spaceSmaller );
     }
   }
-  button {
-    &.MuiIconButton-root,
-    &.MuiButtonBase-root,
-    &.btn,
-    &.tablinks {
-      padding: var( --spaceSmallest ) var( --spaceSmaller );
-      font-size: var( --fontSmall );
-      border: 1px solid var( --controlText15 );
-      color: var( --controlText15 );
-      border-radius: var( --borderRadius1 );
-      display: flex;
-      align-items: center;
-      cursor: pointer;
-      background: var( --surface05 );
-
-      &:hover {
-        background: var( --surface15 );
-      }
+  // button {
```

Comment:
This is the only real change I'm super worried about.  Making the dashboard cards clickable made them buttons, so this style was getting auto applied and they looked bad.